source/plugin/permissions/definitions.rst

```diff
@@ -0,0 +1,181 @@
+    * Only grants the user the permission to teleport to *mynetherworld*.
+
+
+.. note::
+
+    There is no such thing as a ``myPlugin.commands.teleport.worlds.*`` wildcard permission.
+    Use ``myPlugin.commands.teleport.worlds`` for that.
+
+
+PermissionDescription
+=====================
+
+The permission description is an utility that is meant to provide the server owner with details on a certain permission.
+
+The description consists of:
+
+* the target permission id
+* a Text description
+* and an assigned role
+* the owning plugin
+
+If you have a dynamic element such as a world or item type then you can use ``<TemplateParts>``.
+
+Simple-Example
+~~~~~~~~~~~~~~
+
+.. code-block:: text
+    
+    myPlugin.commands.teleport.execute
+    
+    Description: Allows the user to execute the teleport command.
+    Role: user
+    Owner: MyPlugin v1.2.3
+
+Template-Example
+~~~~~~~~~~~~~~~~
+
+.. code-block:: text
+    
+    myPlugin.commands.teleport.worlds.<World>
+    
+    Description: Allows the user to teleport to the world <World>.
+    Role: staff
+    Owner: MyPlugin v1.2.3
+
+.. tip::
+
+    You might skip writing descriptions for some parent permission groups such as ``myPlugin.commands.teleport.worlds``
+    or ``myPlugin.commands`` as their meaning can be derived from the permission structure and the defined children
+    alone.
+
+Subject
+=======
+
+A Subject is a holder of assigned permissions it can be obtained from all CommandSources.
+Permissions can be granted or denied. Plugin authors will usually only deal with Subjects.
+
+Example
+~~~~~~~
+
+The permission plugin's config might look like this.
+
+.. code-block:: text
+    
+    users:
+      Spongie:
+        permissions:
+          - myPlugin.commands.teleport.execute
+          - myPlugin.commands.teleport.others
+          - myPlugin.commands.teleport.worlds
+          - !myPlugin.commands.teleport.worlds.protectedworld
+
+In this case the user `Spongie` was granted the permission to execute the teleport command and teleport her or other players to all worlds, , other than `protectedworld`.
+
+Inheritance
+~~~~~~~~~~~
+
+Subjects also inherit permissions from their parents, unless the given permission is defined in the subject itself.
+Following that rule if any of his parents is granted the ``myPlugin.commands.teleport`` permission the user would
+automatically inherit the ``myPlugin.commands.teleport.all`` permission allowing him to teleport all players at once.
+He still is not able to teleport himself or others to the `protectedworld` though.
+
+If neither the subject itself nor any parent subjects grant or deny a permission then it will be inherited from the
+default subject. The permission plugin/the server owner alone is allowed to define persistent defaults. 
+All other plugins are required to define `transient`/`virtual` defaults for their own permissions.
+
+.. note::
+
+    Persistent permission assignments always take precedence over transient ones.
```

Comment:
Fixed for now. But IMO the written word always have more weight that the transient one.

If a server owner (persistently) defines that a user should not have a specific permission than the transient assignment of a plugin should be ignored.

Could you give me an example in which case it would be better the other way round.